example/server.js

```diff
@@ -15,4 +15,4 @@ const app = express();
 app.use(express.static(__dirname));
 app.use('/graphql', graphqlHTTP(() => ({ schema })));
 
-app.listen(8080, () => console.log('Started on http://localhost:8080/'));
+app.listen(8081, () => console.log('Started on http://localhost:8081/'));
```

Comment:
As this is unrelated, should probably be pulled out into a separate commit.

In any case, I actually think we should use a port number of 0 here, which will cause the OS to assign a random port number. I've just sent a PR for that: https://github.com/graphql/graphiql/pull/321